### PR TITLE
Issue #8990: Checkstyle

### PR DESCRIPTION
**Pull request by aryaniiit002**
Contribution PR: https://github.com/checkstyle/checkstyle/pull/9699
https://github.com/checkstyle/checkstyle/pull/9699#issuecomment-826819142

### CODE REVIEW

File: sevntu-checks/src/test/java/com/github/sevntu/checkstyle/internal/CheckUtil.java [hunk 1]
```diff
@@ -191,7 +191,7 @@ private CheckUtil() {
      * Gets the check message 'as is' from appropriate 'messages.properties'
      * file.
      *
-     * @param locale the locale to get the message for.
+     * @param module the locale to get the message for.
```
Comment (pbludov) [inline sevntu-checks/src/test/java/com/github/sevntu/checkstyle/internal/CheckUtil.java:194]: the module to load resources from.

**Review by pbludov — CHANGES_REQUESTED** [review]
Please fix the misleading parameter description

**Review by rnveach — APPROVED** [review]

File: sevntu-checks/src/test/java/com/github/sevntu/checkstyle/internal/CheckUtil.java (hunk 1 above)
Comment (aryaniiit002) [reply sevntu-checks/src/test/java/com/github/sevntu/checkstyle/internal/CheckUtil.java]: done

**Comment by romani** [comment]
Please squash all commits in one, reference name of check in git commit message

**Comment by aryaniiit002** [comment]
should i send PR from separate branch and not from master?

**Pull request closed without merging**